tests/single-exec/00216.c.tags

```diff
@@ -1,4 +1,3 @@
 portable
-c99
```

Comment:
There were many errors; I didn't try to convert this test to valid C99 as the entire point of it seems to test an extension. An alternative option is to add an 'extension' tag (or, more drastically, just remove the test).